spec/support/api/schemas/song.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://jsonschema.net",
+  "type": "object",
+  "properties": {
+    "id": {
+      "id": "http://jsonschema.net/id",
+      "type": "Object"
```

Comment:
The `"type"` for the ID property should actually be a string, since it's a UUID.  ("Object" is JSON's word for a hash, basically.  So the entire chunk of JSON is an object, but the id itself is a string.  We'll talk more about this stuff on Monday.)